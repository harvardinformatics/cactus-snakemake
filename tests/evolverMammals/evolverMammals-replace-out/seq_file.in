(simHuman_chr6:0.144018,mr:0.271974)Anc1;
simHuman_chr6 evolverMammals-seq/simGorilla.chr6
mr evolverMammals-replace-out/mr.fa
Anc1 evolverMammals-replace-out/Anc1.fa
