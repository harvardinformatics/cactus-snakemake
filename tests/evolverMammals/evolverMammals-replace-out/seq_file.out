(simHuman_chr6:0.144018,mr:0.271974)Anc1;
simHuman_chr6	/n/holylfs05/LABS/informatics/Users/gthomas/cactus-snakemake/tests/evolverMammals/evolverMammals-replace-out/simHuman.chr6
mr	/n/holylfs05/LABS/informatics/Users/gthomas/cactus-snakemake/tests/evolverMammals/evolverMammals-replace-out/mr.fa
Anc1	evolverMammals-replace-out/Anc1.fa
